src/Costume.ts

```diff
@@ -1,22 +1,40 @@
+import { generateId } from "./util/id";
+
+type CostumeDataFormat = "png" | "svg" | "jpeg" | "jpg" | "bmp" | "gif";
```

Comment:
I've never seen `.bmp` costumes before! Does 2.0 allow them?

Reply:
Oh, interesting! This line is taken [from scratch-parser](https://github.com/LLK/scratch-parser/blob/cee1200c858853c0f498d3d01dfbd02f792c6717/lib/sb3_definitions.json#L49-L52). I haven't actually tested bmp costumes in the editor; I'm just matching the official validation format here.